source/ed/build_helper.cpp

```diff
@@ -563,8 +494,8 @@ void builder::build_blocks() {
          }
      }
 
-     for (auto* jp: data->pt_data->journey_patterns) {
-         for (auto& freq_vj: jp->frequency_vehicle_journey_list) {
+     for (auto* route: data->pt_data->routes) {
```

Comment:
Technically, you can, but as you'll modify objects using this path, I prefer letting auto*